### PR TITLE
Add option for default ee from org export

### PR DESCRIPTION
**Pull request by sean-m-sullivan**
# What does this PR do?
Add option for default ee from org export

### CODE REVIEW

File: roles/organizations/tasks/main.yml [hunk 1]
```diff
@@ -6,7 +6,7 @@
     custom_virtualenv:                  "{{ __controller_organizations_item.custom_virtualenv | default(omit, true) }}"
     max_hosts:                          "{{ __controller_organizations_item.max_hosts | default(omit, true) }}"
     instance_groups:                    "{{ __controller_organizations_item.instance_groups | default(omit, true) }}"
-    default_environment:                "{{ (__controller_organizations_item.default_environment | default(omit)) if (assign_default_ee_to_org is defined and assign_default_ee_to_org) else omit }}"
+    default_environment:                "{{ (__controller_organizations_item.default_environment.name | default(__controller_organizations_item.execution_environment | default(omit, true))) if (assign_default_ee_to_org is defined and assign_default_ee_to_org) else omit }}"
```
Comment (djdanielsson) [inline roles/organizations/tasks/main.yml]: Do we want omit true?

Comment (sean-m-sullivan) [reply roles/organizations/tasks/main.yml]: I don't remember anymore, and I was just keeping the original

Comment (djdanielsson) [reply roles/organizations/tasks/main.yml]: I think remove true

**Review by djdanielsson — APPROVED** [review]
LGTM

**Pull request merged**